docs/source/index.rst

```diff
@@ -15,5 +15,6 @@ please visit :doc:`dev`.
    :caption: Documentation Contents
 
    intro
+   nanokbs
    dev
```

Comment:
I noticed today that a '-r' argument is missing in the pip command in the 'dev.rst' file ;)

Reply:
Yes indeed, thanks :+1: